modules/restore-add-master.adoc

```diff
@@ -0,0 +1,101 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/replacing-failed-master.adoc
+
+[id="restore-add-master_{context}"]
+= Adding a master host back to the etcd cluster
+
+Follow these steps to add a master host back to the etcd cluster. This procedure assumes that you previously removed the master host from the cluster and that its etcd dependencies are valid, such as TLS certificates and DNS.
+
+.Prerequisites
+
+* Access to the cluster as a user with the `cluster-admin` role.
+* SSH access to the master host to add to the etcd cluster.
+* The IP address of an existing active etcd member.
+
+.Procedure
+
+. Access the master host to add to the etcd cluster.
++
+[IMPORTANT]
+====
+This procedure _must_ be run on the master host being added to the etcd cluster.
+====
+
+. Run the `etcd-member-add.sh` script:
++
+Pass in two parameters to the `etcd-member-add.sh` script: IP address of an existing etcd member and the name of the etcd member to add.
++
+----
+[core@ip-10-0-147-172 ~]$ sudo -E etcd-member-add.sh 10.0.128.73 etcd-member-ip-10-0-147-172.us-east-2.compute.internal
```

Comment:
Do you need the `[core@ip-10-0-147-172 ~]` prompt here (as well as steps 3-4)?

Reply:
Yeah actually we do, to be clear about which host you are on